src/resources/views/crud/reorder.blade.php

```diff
@@ -65,7 +65,7 @@ function tree_element($entry, $key, $all_entries, $crud)
 
```

Comment:
Ahaa. See, I was surprised `reorder` didn't need the heading classes added. But looks like it did. After we move the heading classes to CSS as I said in https://github.com/Laravel-Backpack/CRUD/pull/4934/files#r1118043859 this page will also look good:

![CleanShot 2023-02-26 at 10 50 03](https://user-images.githubusercontent.com/1032474/221401007-c061cbb7-e0ee-4bf2-b36e-3663583f8407.png)